packages/server/src/main/resources/application.properties

```diff
@@ -145,13 +145,14 @@ terarium.amazon.s3.default.credentials-id=default
 github.url=https://api.github.com
 jsdelivr.url=https://cdn.jsdelivr.net
 mira-api.url=http://mira-epi-dkg-lb-c7b58edea41524e6.elb.us-east-1.amazonaws.com
-mit-proxy.url=https://mit-tr.dev.terarium.ai
-skema-py.url=https://skema-py.dev.terarium.ai
-skema-rs.url=https://skema-rs.dev.terarium.ai
-skema-unified.url=https://skema-unified.dev.terarium.ai
-cosmos-service.url=https://cosmos.dev.terarium.ai
-ciemss-service.url=https://pyciemss.dev.terarium.ai
-climate-data-service.url=https://climate-data.dev.terarium.ai
+mit-proxy.url=https://mit-tr.staging.terarium.ai
```

Comment:
if we are moving these to staging, have you made sure that everything now points to staging?

Reply:
Fixed